kustomize/escalation/base/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- 01-configmap_01-ticketescalation-resources.yaml
+- 01-sonataflow_ticketescalation.yaml
+- 02-configmap_ticketescalation-props.yaml
+
+configurations:
+- config/image-configuration.yaml
+
+  # Remove dev profile from SonataFlow instance(s)
+patches:
+- patch: |
+    - op: remove
+      path: /metadata/annotations/sonataflow.org~1profile
```

Comment:
so this will enable the prod profile by default. however, I'd prefer having an annotation with `prod` value explicitly set since it is reflected better `oc get sonataflow` result (instead of having an empty value for the Profile column. Arguably, this can be reported as an issue for the sonataflow operator)